core-bundle/tests/Controller/ContentElement/ContentElementControllerTest.php

```diff
@@ -206,7 +206,8 @@ public function testSetsTheSharedMaxAgeIfTheElementHasAStartDate(): void
 
         $response = $controller(new Request(), $model, 'main');
 
-        $this->assertSame($expires, $response->getMaxAge());
+        $this->assertGreaterThan($expires - 10, $response->getMaxAge());
```

Comment:
Can't we clock-mock these?

Reply:
I think it should be possible, yes. Because the Controller uses the `time()` function: https://github.com/ausi/contao-contao/blob/927a912b96d98499be6b589c32623434d7a30b17/core-bundle/src/Controller/ContentElement/AbstractContentElementController.php#L46

Reply:
Are you going to adjust the PR accordingly?

Reply:
Yes

Reply:
Done in e7c2f0e9ca5cf4c641b31ea4939ce929c770a0a4